README.md

```diff
@@ -107,7 +106,7 @@ device.getVolume()
   * getLEDState()
   * getMusicLibrary(search, options)
   * getMuted()
-  * getTopology()
+  * ~~getTopology()~~ Doesn't work if you upgraded to Sonos v9.1 
```

Comment:
Could another note be added suggesting the alternative function?

Reply:
Added the suggestion